src/abstracts/SablierV2MerkleStreamer.sol

```diff
@@ -53,13 +60,19 @@ abstract contract SablierV2MerkleStreamer is
     constructor(
         address initialAdmin,
         IERC20 asset,
+        string memory name,
         bytes32 merkleRoot,
         uint40 expiration,
         bool cancelable,
         bool transferable
     ) {
+        if (bytes(name).length > 32) {
```

Comment:
I had the following reason to place this check in `SablierV2MerkleStreamer`:

We still need this check in `SablierV2MerkleStreamerLL` since that contract can independently be deployed. So ideally, the check on name length should be a part of this contract. However, since we import `Errors` in `SablierV2MerkleStreamer`, I decided to keep the check in `SablierV2MerkleStreamer` to avoids re-importing `Errors` in `SablierV2MerkleStreamerLL`.

I am happy to keep the check in `SablierV2MerkleStreamerFactory` as well as `SablierV2MerkleStreamerLL` but then it will just repeat itself.